homeassistant-supervised/etc/systemd/resolved.conf

```diff
@@ -1,25 +0,0 @@
-#  This file is part of systemd.
-#
-#  systemd is free software; you can redistribute it and/or modify it
-#  under the terms of the GNU Lesser General Public License as published by
-#  the Free Software Foundation; either version 2.1 of the License, or
-#  (at your option) any later version.
-#
-# Entries in this file show the compile time defaults.
-# You can change settings by editing this file.
-# Defaults can be restored by simply deleting this file.
-#
-# See resolved.conf(5) for details
-
-[Resolve]
-#DNS=
-#FallbackDNS=1.1.1.1 8.8.8.8 1.0.0.1 8.8.4.4 2606:4700:4700::1111 2001:4860:4860::8888 2606:4700:4700::1001 2001:4860:4860::8844
-#Domains=
-DNSSEC=no
-DNSOverTLS=no
```

Comment:
These two configs got most likely copied over from the HAOS config at https://github.com/home-assistant/operating-system/commit/e3120df5d08da5aade13e5bcbf9a094cd538fde3.

Buildroot (used to build HAOS) by default sets `DNSOverTLS` to opportunistic mode, which did cause issues in certain cases. Hence we opt out of it. Debian uses the systemd default configuration which is off.

For DNSSEC, Debian explicitly turns it off by default.

So it is fairly safe to not set these two options, as their default is `no` already.